src/cmdc/datasets/covidactnow/data.py

```diff
@@ -65,23 +66,26 @@ def get(self):
             df[c] = pd.to_datetime(df[c])
 
         df["fips"] = df["fips"].astype(int)
-
-        return df
+        return df.melt(id_vars=self.idx)
 
     def _insert_query(self, df, table_name, temp_name, pk):
         cols = list(df)
         intervention_ix = cols.index("intervention")
         final_cols = cols.copy()
         final_cols[intervention_ix] = "intervention_id"
+        variable_ix = cols.index("variable")
+        final_cols[variable_ix] = "variable_id"
 
         temp_cols = list(df)
         temp_cols[intervention_ix] = "it.id as intervention_id"
+        temp_cols[variable_ix] = "vt.id as variable_id"
 
         out = f"""
         INSERT INTO data.{table_name} ({", ".join(final_cols)})
         SELECT {", ".join(temp_cols)}
         from {temp_name} tt
         LEFT JOIN meta.actnow_intervention_types it on it.name = tt.intervention
+        LEFT JOIN meta.{self.table_name + "_variables"} vt on vt.name = tt.variable
```

Comment:
I see -- The column names probably come directly from CAN data which you can just use...

Disregard both comments. This all looks good.